src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { expect } from 'chai';
+import userEvent from '@testing-library/user-event';
+
+import { mockFetch, resetFetch } from 'platform/testing/unit/helpers';
+
+import {
+  createTestStore,
+  renderWithStoreAndRouter,
+  setTypeOfCare,
+  setTypeOfFacility,
+} from '../../../mocks/setup';
+import { getParentSiteMock } from '../../../mocks/v0';
+import {
+  mockCommunityCareEligibility,
+  mockParentSites,
+} from '../../../mocks/helpers';
+
+import CommunityCareProviderSelectionPage from '../../../../new-appointment/components/CommunityCareProviderSelectionPage';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingCommunityCare: true,
+  },
+  user: {
+    profile: {
+      facilities: [{ facilityId: '983', isCerner: false }],
+    },
+  },
+};
+describe('VAOS <CommunityCareProviderSelectionPage>', () => {
+  beforeEach(() => mockFetch());
+  afterEach(() => resetFetch());
+  it('should display closest city question when user has multiple supported sites', async () => {
```

Comment:
Copy of one of the tests from the other page, so we have something testing the new page we can plug new tests into